app/assets/stylesheets/style.scss

```diff
@@ -446,14 +352,26 @@ label:hover {
   width: 100%;
 }
 
+.splash-tinder {
+  position: absolute;
+  left: 20%;
+}
+
 #tinderslide {
   width: 1000px;
   height: 550px;
   margin: auto;
   background-color: $background;
+  margin-top: 100px;
 
   img {
     float: left;
+    margin: 20px 20px 0 20px;
+    border-radius: 10px;
+
+    -webkit-box-shadow: 2px 2px 15px 2px #a2a2a2;
+    -moz-box-shadow: 2px 2px 15px 2px #a2a2a2;
```

Comment:
Color literals like `#a2a2a2` should only be used in variable declarations; they should be referred to via variable everywhere else.